RPi/install_dashboard.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ "$(id -u)" -ne 0 ]; then
+        echo 'This script must be run by root' >&2
+        exit 1
+fi
+
+arch=arm64
```

Comment:
Maybe use $(uname -m) insted?

Reply:
This returns "aarch64" on 64bit arm. I could set arch explicitly on a condition of this return, which at least makes it automatic.